glide.yaml

```diff
@@ -1,25 +1,18 @@
 package: go.uber.org/fx
 import:
-- package: github.com/go-validator/validator
-- package: github.com/ogier/pflag
-  version: ~0.0.1
 - package: github.com/pkg/errors
   version: ~0.8.0
 - package: go.uber.org/multierr
   version: ~0.2.0
-- package: gopkg.in/yaml.v2
 - package: go.uber.org/dig
   # TODO: cut v0.4 of dig, and get ready to solidify v1.0
   version: master
 testImport:
-- package: github.com/google/gofuzz
 - package: github.com/stretchr/testify
   version: ~1.1.4
   subpackages:
   - assert
   - require
-- package: go.uber.org/zap
```

Comment:
fx does not rely or has anything to do with zap.
But you bring a good point, why does config need zap? Lets open a PR in the config repo to remove zap usage